### PR TITLE
e2e for cdc

### PR DESCRIPTION
**Pull request by weekface**
### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: tests/e2e/util/image/image.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ const (
 	TiDBV3UpgradeVersion          = "v3.0.9"
 	TiDBV4Version                 = "v4.0.0-rc"
 	TiDBV4UpgradeVersion          = "v4.0.0-rc.2"
+	TiDBNightly                   = "nightly"
```
Comment (cofyc) [inline tests/e2e/util/image/image.go:34]: Append this constant to `images` in `ListImages` function

File: tests/e2e/tidbcluster/tidbcluster.go [hunk 2]
```diff
@@ -1273,6 +1274,52 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
...
+
+		ginkgo.By("Creating change feed task")
+		fromTCName := fromTc.Name
+		toTCName := toTc.Name
+		changeFeedCMD := fmt.Sprintf("/cdc cli changefeed create "+
+			"--sink-uri=\"tidb://root:@%s:4000/\" --pd=http://%s:2379",
+			controller.TiDBMemberName(toTCName), controller.PDMemberName(fromTCName))
+		cmd := fmt.Sprintf("kubectl exec -n %s %s-0 -- %s",
+			ns, controller.TiCDCMemberName(fromTCName), changeFeedCMD)
+		data, err := exec.Command("sh", "-c", cmd).CombinedOutput()
```
Comment (cofyc) [inline tests/e2e/tidbcluster/tidbcluster.go:1310]:
```suggestion
		data, err := framework.RunKubectl("exec", "-n", ns, controller.TiCDCMemberName(fromTCName), "--", changeFeedCMD)
```

Comment (cofyc) [inline tests/e2e/tidbcluster/tidbcluster.go:1310]:
```suggestion
		changeFeedCMD := fmt.Sprintf("/cdc cli changefeed create "+
			"--sink-uri=\"tidb://root:@%s:4000/\" --pd=http://%s:2379",
			controller.TiDBMemberName(toTCName), controller.PDMemberName(fromTCName))
		cmd := fmt.Sprintf("kubectl exec -n %s %s-0 -- %s",
			ns, controller.TiCDCMemberName(fromTCName), changeFeedCMD)
		data, err := framework.RunKubectl("exec", "-n", ns, controller.TiCDCMemberName(fromTCName) + "-0", "--", "/cdc", "cli", "changefeed", "create", fmt.Sprintf("--sink-uri=tidb://root:@%s:4000/", controller.TiDBMemberName(toTCName)), fmt.Sprintf("--pd=http://%s:2379", controller.PDMemberName(fromTCName))
```

`framework.RunKubectl` is a convenient wrapper to exec raw kubectl command. With it, we don't need to care about underlying details.

Comment (Yisaer) [reply tests/e2e/tidbcluster/tidbcluster.go:1310]: cooool.

**Pull request merged**